src/main.js

```diff
@@ -25,18 +31,21 @@ try {
   const end_time = `${format(endDate, 'yyyy-MM-dd')}T${format(endDate, 'HH:mm:sszzzz')}Z`;
   core.info(`Window will be open from ${start_time} -> ${end_time}`);
 
+  serviceIds = [serviceIdInput]
+    .concat(serviceIdsInput.split(','))
+    .filter(serviceId => serviceId)
+    .map(serviceId => ({
+      id: serviceId.trim(),
+      type: 'service'
+    }));
```

Comment:
From what I could tell it doesn't look like serviceIds is declared anywhere.  I don't think the filter call is necessary either.
```suggestion
const serviceIds = [serviceIdInput]
    .concat(serviceIdsInput.split(','))
    .map(serviceId => ({
      id: serviceId.trim(),
      type: 'service'
    }));
```

Reply:
The fitler is necessary.  It is filtering out empty strings and null.

Reply:
Your right, I forgot to declare `serviceIds`.  Good catch.